src/components/TicketDetails/TicketDetails.tsx

```diff
@@ -0,0 +1,72 @@
+import { Box, Container, Typography } from 'components'
+
+import { FC } from 'react'
+
+import * as S from './TicketDetails.styled'
+
+export type TicketDetailsProps = {
+  eventName: string
+  startDate: string
+  endDate: string
+  startTime: string
+  endTime: string
+  qrCode: string
+  location: string
+}
+
+export const TicketDetails: FC<TicketDetailsProps> = ({
+  eventName,
+  startDate,
+  startTime,
+  endDate,
+  endTime,
+  location,
+  qrCode,
+}) => {
+  return (
```

Comment:
no need in `return` keyword
```suggestion
}) => (
  <Container ...
```